com.unity.shadergraph/Editor/GraphUI/ShaderGraphRegistryBuilder.cs

```diff
@@ -20,22 +24,45 @@ public static Registry CreateDefaultRegistry(Action<RegistryKey, Type> afterNode
             reg.Register<GradientNode>();
             reg.Register<SampleGradientNode>();
             reg.Register<ShaderGraphContext>();
-            //RegistryInstance.Register<Registry.Types.AddNode>();
 
-            // Register nodes from FunctionDescriptors in IStandardNode classes.
+            // Register nodes from IStandardNode implementers.
             var interfaceType = typeof(IStandardNode);
             var types = AppDomain.CurrentDomain.GetAssemblies()
                 .SelectMany(s => s.GetTypes())
                 .Where(p => interfaceType.IsAssignableFrom(p));
-            string m = "get_FunctionDescriptor";
-            foreach (var t in types)
+            var tArr = types.ToArray();
+
+            Debug.LogWarning(tArr);
```

Comment:
Not sure if you intended to leave this in, seems fine either way

Reply:
Thank you for catching this. I left this `Debug` in by mistake.

Reply:
Change made.